File: keycloak-client/src/main/java/cc/coopersoft/accesstoken/interceptor/AllAccessTokenInterceptor.java
package cc.coopersoft.accesstoken.interceptor;

import cc.coopersoft.accesstoken.api.AccessTokenHolder;
import okhttp3.Response;

import java.io.IOException;

public class AllAccessTokenInterceptor extends AccessTokenInterceptor {

  public AllAccessTokenInterceptor(AccessTokenHolder tokenHolder) {
    super(tokenHolder);
  }

  @Override
  public Response intercept(Chain chain) throws IOException {
    return accessTokenRequest(chain);
  }
}

File: keycloak-client/src/main/java/cc/coopersoft/accesstoken/interceptor/AllAccessTokenInterceptor.kt
package cc.coopersoft.accesstoken.interceptor

import cc.coopersoft.accesstoken.api.AccessTokenHolder
import cc.coopersoft.accesstoken.interceptor.AccessTokenInterceptor
import okhttp3.Interceptor
import okhttp3.Response
import java.io.IOException

class AllAccessTokenInterceptor(tokenHolder: AccessTokenHolder?) : AccessTokenInterceptor(
    tokenHolder!!
) {
    @Throws(IOException::class)
    override fun intercept(chain: Interceptor.Chain): Response {
        return accessTokenRequest(chain)
    }
}